web_src/js/components/RepoBranchTagSelector.vue

```diff
@@ -110,12 +114,16 @@ const sfc = {
     formActionUrl() {
       return `${this.repoLink}/branches/_new/${this.branchNameSubURL}`;
     },
+    createTag() {
+      return this.mode === 'tags';
```

Comment:
I expected a different function name given this implementation.
What about i.e. `shouldCreateTag`?
(I know, will change some previously unaffected parts too)

Reply:
Rename done in https://github.com/go-gitea/gitea/pull/25719/commits/5c5ac0aaa023b369ed41c11ea2ba21ce630637fa